src/gpl/src/routeBase.cpp

```diff
@@ -868,16 +877,16 @@ float RouteBase::getRudyRC() const
   avg020RC /= ceil(0.020 * arraySize);
   avg050RC /= ceil(0.050 * arraySize);
 
-  log_->info(GPL, 83, "0.5%RC: {:.4f}", avg005RC);
-  log_->info(GPL, 84, "1.0%RC: {:.4f}", avg010RC);
-  log_->info(GPL, 85, "2.0%RC: {:.4f}", avg020RC);
-  log_->info(GPL, 86, "5.0%RC: {:.4f}", avg050RC);
+  log_->info(GPL, 83, "Average top 0.5% routing congestion: {:.4f}", avg005RC);
+  log_->info(GPL, 84, "Average top 1.0% routing congestion: {:.4f}", avg010RC);
+  log_->info(GPL, 85, "Average top 2.0% routing congestion: {:.4f}", avg020RC);
+  log_->info(GPL, 86, "Average top 5.0% routing congestion: {:.4f}", avg050RC);
 
   float finalRC = (rbVars_.rcK1 * avg005RC + rbVars_.rcK2 * avg010RC
                    + rbVars_.rcK3 * avg020RC + rbVars_.rcK4 * avg050RC)
                   / (rbVars_.rcK1 + rbVars_.rcK2 + rbVars_.rcK3 + rbVars_.rcK4);
 
-  log_->info(GPL, 87, "FinalRC: {}", finalRC);
+  log_->info(GPL, 87, "Final routing congestion: {}", finalRC);
```

Comment:
`Final` is unclear.  How about `Weighted routing congestion`?  My recollection is that most of these weights are actually zero and the whole thing could be simplified but that can be another PR.